superset/assets/src/visualizations/FilterBox/FilterBox.jsx

```diff
@@ -123,7 +123,7 @@ class FilterBox extends React.Component {
               label={t('Time range')}
               description={t('Select start and end date')}
               onChange={(...args) => { this.changeFilter(TIME_RANGE, ...args); }}
-              value={this.state.selectedValues[TIME_RANGE]}
+              value={this.state.selectedValues[TIME_RANGE] || 'No filter'}
```

Comment:
Is `No filter` a compatible value for `DateFilterControl`? I haven't run this but notice the `DateFilterControl` will attempt to parse `value`. Not sure how it would deal with this.
Just don't want it to become `{since: 'No', until: 'filter' }`

Reply:
'No filter' is one of the `COMMON_TIME_FRAMES` so it won't be parsed and split into since/until incorrectly. I'm a little worried about this not being robust (if the value changes then this will break), but I'm not sure we want to change the default value for the DateFilterControl to be 'No filter'. I added the test so if this gets changed it will break.